sotodlib/preprocess/processes.py

```diff
@@ -1266,6 +1316,31 @@ def calc_and_save(self, aman, proc_aman):
             # planet_aman = core.AxisManager(core.OffsetAxis("samps", onsamp))
             planet_aman.wrap("xi_p", xi_p, [(0, "ds_samps")])
             planet_aman.wrap("eta_p", eta_p, [(0, "ds_samps")])
+            if telescope == 'SAT':
+                for ws in wafer_slots:
+                    if ws in streamed_wafer_slots:
+                        ws_dets = np.where(aman.det_info.wafer_slot == ws)[0]
+                        ws_xi = aman.focal_plane.xi[ws_dets]
+                        ws_eta = aman.focal_plane.eta[ws_dets]
+                        wafer_hit = np.sum([np.any(np.sqrt((xi_p-xi)**2 + (eta_p-eta)**2) < np.deg2rad(0.5)) for xi, eta in zip(ws_xi, ws_eta)]) > self.calc_cfgs.get("wafer_hit_threshold", 10)
+                        if wafer_hit:
+                            planet_aman.wrap(ws, True)
+                        else:
+                            planet_aman.wrap(ws, False)
+                    else:
+                        planet_aman.wrap(ws, False)
+            elif telescope == 'LAT':
+                for ws in wafer_slots:
+                    ws_center_xi = np.deg2rad(wafer_centers[ws][0])
+                    ws_center_eta = np.deg2rad(wafer_centers[ws][1])
+                    wafer_hit = np.sum([np.sqrt((xi_p-ws_center_xi)**2 + (eta_p-ws_center_eta)**2) < np.deg2rad(0.5)]) > self.calc_cfgs.get("wafer_hit_threshold", 10)
+                    if wafer_hit:
```

Comment:
So for each obs, the hit stat is recorded _for all tube-wafer_ pairs?  i.e. even for the other LAT tubes?